interface/forms/care_plan/new.php

```diff
@@ -45,15 +45,6 @@
         <title><?php echo xlt("Care Plan Form"); ?></title>
```

Comment:
Doesn't look like you added the container correctly here still.
![image](https://user-images.githubusercontent.com/46497068/91076765-f81f8500-e60d-11ea-9935-a1626ac8c076.png)
Trying to get rid of this issue here.